userguide/system.rst

```diff
@@ -2151,14 +2155,14 @@ notifications as the reports are actioned.
 Before creating a bug report or feature request, ensure that an
 existing report does not already exist at
 https://redmine.ixsystems.com/projects/freenas/issues.
-If you find a similar issue that is not yet marked as *closed* or
-*resolved*, add a comment to that issue if you have new information
-to provide that can assist in resolving the issue. If you find a
-similar issue that is marked as *closed* or *resolved*, you can
+If there is a similar issue that is not yet marked as *closed* or
```

Comment:
Rewrite to make this a less wandering sentence.

If a similar issue is already present and has not been marked *Closed* or *Resolved*,
comment on that issue, adding new information to help solve it. If similar issues have
already been closed or resolved, create a new issue and refer to the previous issue.